packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -22,27 +22,46 @@ export const Checkbox = compose<ICheckboxType>({
       console.warn('defaultChecked and checked are mutually exclusive to one another. Use one or the other.');
     }
 
-    // Used for uncontrolled Checkbox's to keep internal state
-    const data = useAsToggle(defaultChecked, checked, onChange);
+    // Re-usable hook for toggle components.
+    const [isChecked, toggleChecked] = useAsToggle(defaultChecked, checked, onChange);
 
-    const pressable = useAsPressable({ onPress: data.onChange, ...rest });
+    const pressable = useAsPressable({ onPress: toggleChecked, ...rest });
 
     const buttonRef = useViewCommandFocus(userProps.componentRef);
 
     // Handles the "Space" key toggling the Checkbox
-    const onKeyUpSpace = useKeyCallback(' ', data.onChange);
+    const onKeyUpSpace = useKeyCallback(' ', toggleChecked);
```

Comment:
does (should? spec question) enter also toggle the checkbox?

Reply:
Nope, neither Fabric or Win32 Checkbox's toggle on Enter :)